examples/predicates/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+#[cfg(test)]
+mod tests {
+    use fuels::prelude::Error;
+
+    #[tokio::test]
+    async fn predicate_example() -> Result<(), Error> {
+        use fuels::contract::predicate::Predicate;
+        use fuels::prelude::*;
+        use fuels::signers::fuel_crypto::SecretKey;
+
+        // ANCHOR: predicate_wallets
+        let secret_key1: SecretKey =
```

Comment:
A fun helper:
```rust
    fn wallets_from_keys<const T: usize>(keys: &[&str; T]) -> [LocalWallet; T] {
        keys.map(|hash| LocalWallet::new_from_private_key(hash.parse().unwrap(), None))
    }
    // ... later on
        let [wallet, wallet2, wallet3] = wallets_from_keys(&[
            "0x862512a2363db2b3a375c0d4bbbd27172180d89f23f2e259bac850ab02619301",
            "0x37fa81c84ccd547c30c176b118d5cb892bdb113e8e80141f266519422ef9eefd",
            "0x976e5c3fa620092c718d852ca703b6da9e3075b9f2ecb8ed42d9f746bf26aafb",
        ]);

```

Reply:
Albeit only now do I see the anchors. We probably keep things 'vanilla' here :) 

But I'll leave the comment be :) Might be useful in the future.